src/library/driftory.ts

```diff
@@ -44,7 +44,7 @@ export interface DriftoryArguments {
    * To prevent this, you can use this parameter to provide your own instance of OpenSeaDragon instead.
    *  */
   OpenSeadragon?: OpenSeadragonType;
-  /** Called whenever driftory navigates to a new frame (excludes when the user drags manually). */
+  /** Called whenever driftory navigates to a new frame, whether via clicking, dragging, keys, or API. */
```

Comment:
Oh cool! I didn't realise that it also triggered on mouse drag. I thought that would have been too difficult to calculate 😊